tools/tcat_ble_client/tlv/tcat_tlv.py

```diff
@@ -47,7 +47,14 @@ class TcatTLVType(Enum):
     GET_ACTIVE_DATASET = 0x40
     GET_DIAGNOSTIC_TLVS = 0x26
     DECOMMISSION = 0x60
-    APPLICATION = 0x82
+    GET_APPLICATION_LAYERS = 0x80
+    APPLICATION_1 = 0x81
```

Comment:
For all these TLV names here, we should base it on the actual spec TLV names as much as possible. So e.g.

```suggestion
    APPLICATION_DATA_1 = 0x81
```

Reply:
fixed.